### ISSUE TITLE
Octernship Assignment Deliverables ?

### ISSUE THREAD

**Issue by AbhishekRP2002** [T3xtifyai/Octernship#1]
The task expects the participant to identify a unique monetization opportunity with B2C AI applications. I am confused about what we have to deliver finally for the task submission, will it be a full-fledged project involving all the tech stacks necessary or will it be just documentation representing our ideas on how we can think of ways to monetize B2C AI solutions?

**Comment by Jayakrishna2b** [comment]
i think participant need to give  B2C ai application idea which can be monetized

**Comment by Armandotrsg** [comment]
I agree, the instructions are unclear of how we should deliver the assignment, and I am also confused if we should be chosen to participate and then do the assignments or we should do the assignment to be chosen.

**Comment by CharanMN7** [comment]
Once you click "apply" for the octernship, it will be added to "Your Projects". You'll see a link called "Introduction". This will lead you to a repository called "introduction-YOUR_GITHUB_USERNAME". This is where you are supposed to submit your work through an auto-generated pull request.

As for the deliverables, they can be anything like a presentation, a document, or a video. The assignment is to ideate and present ways in which one can monetize the B2C AI solutions.

**Comment by davidemusi** [comment]
Have followed the steps highlighted and submitted a presentation

**Comment by davidemusi** [comment]
After submission, what next?

**Comment by Susmit-Ullas** [comment]
In the introduction to the task for the Textify Octernship, the task submission section says "Students are expected to use the GitHub Flow when working on their project. Making changes on the auto generated feedback branch to complete the task Using the auto generated Feedback Pull Request for review and submission Using GitHub Discussions to ask any relevant questions regarding the project".

The "auto generated feedback branch" and "auto generated Feedback Pull Request" has not yet been generated, the deadline for submitting the task is April 1, 2023. The auto generated branches and pull requests for other Octernship tasks have already been generated and their deadline is May 15, 2023.

What should I do? Should I commit changes by making my own branch and pull request and submit the task? Or is there anything else I should do?

**Comment by AbhishekRP2002** [comment]
Did anyone receive the acceptance mail?
